packages/zowe-explorer/src/webviews/src/edit-history/components/PersistentTable/PersistentTableData.tsx

```diff
@@ -35,7 +35,7 @@ export default function PersistentTableData({ persistentProp }: Readonly<{ persi
   };
 
   const renderSelectButton = (item: string, i: number) => {
-    return selection[type] === "search" || selection[type] === "fileHistory" ? (
+    return selection[type] === "search" || selection[type] === "fileHistory" || selection[type] === "encodingHistory" ? (
```

Comment:
If you go ahead and do the refactor suggested by @zFernand0 , might be good to add it here as well!